packages/tokens/config.js

```diff
@@ -83,6 +83,14 @@ module.exports = {
           destination: 'js/themes.js',
           format: 'javascript/es6/vads-colors-themes',
         },
+        {
+          destination: 'js/font/styles.js',
+          format: 'javascript/es6/simple-key-value',
+          filter: 'filter/font/composite-npm',
+          options: {
+            exportName: 'styles',
```

Comment:
```suggestion
            exportName: 'typography',
```
I'd always been under the impression using the tokens directly would be `typography.whatever` given they're the typography tokens.

Reply:
Also suggest updating the destination files accordingly.